.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,65 @@
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          args: >
+            -Dsonar.host.url=https://sonarcloud.io
+            -Dsonar.projectKey=Learnathon-By-Geeky-Solutions_triangle
+            -Dsonar.organization=learnathon-by-geeky-solutions
+            -Dsonar.sources=lib/
+            -Dsonar.tests=test/
+            -Dsonar.java.version=17
+            -Dsonar.verbose=true
+            -Dsonar.dart.coverage.reportPath=coverage/lcov.info
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)